Resources/config/body_listener.xml

```diff
@@ -15,7 +15,7 @@
         <service id="fos_rest.decoder.xml" class="FOS\RestBundle\Decoder\XmlDecoder" />
 
         <service id="fos_rest.decoder_provider" class="FOS\RestBundle\Decoder\ContainerDecoderProvider">
-            <argument>%fos_rest.decoders%</argument>
+            <argument /> <!-- decoders -->
```

Comment:
I would use ``<argument type="collection">`` here, so that the non-replaced argument is already an array